### PR TITLE
LWIMIQ - fixing bugs

### PR DESCRIPTION
**Pull request by ZitaNemeckova**
New blogpost `LWIMIQ - fixing bugs`

### CODE REVIEW

**Comment by JPrause** [comment]
@jntullo @imtayadeway @syncrou can you each review if time permits.

**Comment by JPrause** [comment]
@jrafanie you might also want to review. Zita talks about your favorite subject.

**Comment by JPrause** [comment]
LGTM 👍 thanks @ZitaNemeckova

File: site/_posts/2017-12-05-last-week-in-manageiq-fixing-bugs.html.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+tags: LWIMIQ
+---
+
+Hi. I, [ZitaNemeckova](https://github.com/ZitaNemeckova) will tell you more about our effort to improve our codebase. It's not only about fixing code but also about cleaning and deleting. If you never used static analysis tool to find dead methods this post is for you.
+
+## Deleted
+
+Let's take a small detour and talk about one of our community activities for a bit. Did you know that part of Brno team is collaborating with Mendel University in Brno on teaching students how to contribute to Open Source? 
+
+Probably not as it's in prototyping phase. Last year they helped with conversion of forms in Angular and wrote some tests in Ruby. This year they started our favorite activity: deleting code.
```
Comment (romanblanco) [inline site/_posts/2017-12-05-last-week-in-manageiq-fixing-bugs.html.md]: Maybe `Last year they helped` to `Last year the students helped` if it sounds better 😉

File: site/_posts/2017-12-05-last-week-in-manageiq-fixing-bugs.html.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Deleted
+
+Let's take a small detour and talk about one of our community activities for a bit. Did you know that part of Brno team is collaborating with Mendel University in Brno on teaching students how to contribute to Open Source? 
+
+Probably not as it's in prototyping phase. Last year they helped with conversion of forms in Angular and wrote some tests in Ruby. This year they started our favorite activity: deleting code. 
+
+It's known that from time to time a method is no longer called but nobody notices so it stays as a zombie that hides among the living code. Haunting the code :ghost: . 
+
+It's easy to run some static analysis tool like [Debride](https://github.com/seattlerb/debride) that can find all those hiding zombies in [Classic UI](https://github.com/ManageIQ/manageiq-ui-classic). But it's so good it also points out methods that are living a bit different lives. They are called dynamically :godmode:
```
Comment (jntullo) [inline site/_posts/2017-12-05-last-week-in-manageiq-fixing-bugs.html.md]: Great info about Debride! One minor nit, perhaps the "living a bit different lives" could be clarified with something along the lines of "methods that are living a bit of a different life" ?

**Comment by syncrou** [comment]
@ZitaNemeckova - I really enjoyed the Detour!  Great article.  👍

**Comment by ZitaNemeckova** [comment]
Thanks everyone :) All comments are resolved.

**Comment by JPrause** [comment]
@cybette seems we are ready for your last look over before merge.

File: site/_posts/2017-12-05-last-week-in-manageiq-fixing-bugs.html.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+author: zita
+date: 2017-12-06
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi. I, [ZitaNemeckova](https://github.com/ZitaNemeckova) will tell you more about our effort to improve our codebase. It's not only about fixing code but also about cleaning and deleting. If you never used static analysis tool to find dead methods this post is for you.
+
+## Deleted
```
Comment (imtayadeway) [inline site/_posts/2017-12-05-last-week-in-manageiq-fixing-bugs.html.md]: Nice! Love that this is front and center! 😁

File: site/_posts/2017-12-05-last-week-in-manageiq-fixing-bugs.html.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+Let's take a small detour and talk about one of our community activities for a bit. Did you know that part of Brno team is collaborating with Mendel University in Brno on teaching students how to contribute to Open Source? 
+
+Probably not as it's in prototyping phase. Last year students helped with conversion of forms in Angular and wrote some tests in Ruby. This year they started our favorite activity: deleting code. 
+
+It's known that from time to time a method is no longer called but nobody notices so it stays as a zombie that hides among the living code. Haunting the code :ghost:. 
+
+It's easy to run some static analysis tool like [Debride](https://github.com/seattlerb/debride) that can find all those hiding zombies in [Classic UI](https://github.com/ManageIQ/manageiq-ui-classic). But it's so good it also points out methods that are living a bit different life. They are called dynamically :godmode:.
+  
+Then it's good time to use our 40 students to brute force hunting down zombies amongst 300 possibly dead methods. And they were able to single out 50 zombies to be beheaded in the following weeks.
```
Comment (jrafanie) [inline site/_posts/2017-12-05-last-week-in-manageiq-fixing-bugs.html.md]: ✂️ 🔥 ✂️ 🔥 ✂️ 🔥 

🍰  🍪 👏 🙇 😍 🎉

**Pull request merged**